sys/include/net/gcoap.h

```diff
@@ -270,7 +277,8 @@ extern "C" {
 #define GCOAP_HEADER_MAXLEN     (sizeof(coap_hdr_t) + GCOAP_TOKENLEN_MAX)
 
 /**
- * @brief   Length in bytes for a token; use 2 if not defined
+ * @ingroup net_gcoap_conf
+ * @brief   Length in bytes for a token
```

Comment:
Is this supposed to be in any relation to `GCOAP_TOKENLEN_MAX`? If yes please document.

Reply:
Good catch. Documented in 03d4230.